package.json

```diff
@@ -100,6 +100,11 @@
 					},
 					"description": "The paths to be treated as server-sided directories. Overrides rojo if enabled."
 				},
+				"roblox-ts.status.show": {
```

Comment:
I think it'd be better if this was `roblox-ts.command.status.show` so that it's grouped together with the relevant settings, since unfortunately vscode doesn't allow us to order them manually.

Reply:
I think so too